frontend/src/app/libs/common-components/components/event-tile/event-tile.component.html

```diff
@@ -0,0 +1,55 @@
+<section class="app-event-tile" 
+		[class.app-event-tile--draft]="isDraft">
+		
+	<div class="app-event-tile__container">
+		<h3 class="app-event-tile__title" [attr.title]="name"> {{ name }} </h3>
+
+		<ng-container *ngIf="endDate else inProgressTime">
+			<div class="app-event-tile__date-time">
+				<time class="app-event-tile__date">
+					<span>{{ startDate | date:'d MMM yy' }}</span>
```

Comment:
If possible, please, move these formats in component and use just one variable in 2 places